docs/user-guide/kubernetes.md

```diff
@@ -100,6 +100,16 @@ curl $(minikube ip)
 
 > We expect to see a 404 response here as we haven't yet given Træfɪk any configuration.
 
+## Deploy Træfɪk using Helm Chart
+
+Install Træfɪk chart by:
```

Comment:
Could you put it somewhat of a transition from the previous section by saying

_"Instead of installing Traefik via a Deployment object, you can also use the Traefik Helm chart."_